RPIPs/RPIP-47.md

```diff
@@ -33,8 +33,8 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
   - Setting the previous `megapool delegate`'s expiration block to occur `delegate_upgrade_buffer` after the upgrade
 - A node operator SHALL be able to upgrade their active `megapool delegate` to the latest `megapool delegate` at any time (including after the active `megapool delegate`'s expiration block)
 - After the active `megapool delegate`'s expiration block:
-  - The active `megapool delegate` SHALL NOT be usable by the node operator
-  - The node operator SHALL NOT be able to claim any rewards
+  - Functions in the active megapool delegate SHALL revert when not called by a protocol contract
+  - No capital SHALL be distributed to the withdrawal address(es)
```

Comment:
```suggestion
  - Node operator capital SHALL be distributed to the withdrawal address(es)
```

Reply:
I assumed by "No" you meant "NO" short for Node operator... Generally I wonder if I should sweep through all the RPIPs and change any "NO" to say node operator...

Reply:
Also should this be made a permissionless function (in which case maybe that should go first, and then "all other functions in the active megapool delegate SHALL revert..."? Or best to just require protocol contract to distribute rewards?

Reply:
Quite the opposite -- I meant that no capital (aka zero capital) SHALL be distributed